UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartTraumaDamage.cs

```diff
@@ -81,18 +81,36 @@ public partial class BodyPart
 
 		/// <summary>
 		/// Does this body part have a cut and how big is it?
-		/// </summary>
 		private BodyPartCutSize currentCutSize = BodyPartCutSize.NONE;
 
 		public bool CanBleedInternally = false;
 
 		public bool CanBleedExternally = false;
 
+		public bool CanBeBroken = false;
+
+		private bool isFractured_Compound = false;
+		private bool isFractured_Hairline = false;
+		private bool jointDislocated = false; //TODO : ADD LATER.
+
+		/// <summary>
+		/// Critcal Blunt Trauma damage.
+		/// </summary>
+		public bool IsFractured_Compound => isFractured_Compound;
```

Comment:
why the underscore?

Reply:
I found that putting the level name after IsFreacutred to be much more readable instead of `IsFracturedJointDislocation`